Explorer/Assets/Scripts/ECS/Unity/Materials/Systems/ResetMaterialSystem.cs

```diff
@@ -22,7 +22,7 @@ public partial class ResetMaterialSystem : BaseUnityLoopSystem
         private readonly DestroyMaterial destroyMaterial;
         private readonly ISceneData sceneData;
 
-        internal ResetMaterialSystem(World world, DestroyMaterial destroyMaterial, ISceneData sceneData) : base(world)
+        public ResetMaterialSystem(World world, DestroyMaterial destroyMaterial, ISceneData sceneData) : base(world)
```

Comment:
why do you need to have it `public`? I assume it is for testing, but then `AssemblyInfo.cs` should help with it